From b8ad862eca37c1b3cbfa19d3e2a419d09c566533 Mon Sep 17 00:00:00 2001
From: beste <1596775839@qq.com>
Date: Fri, 24 May 2024 16:33:18 +0800
Subject: [PATCH] FEAT:Reduce CopyrightNotice text size

---
 CopyrightNoticeDialog.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CopyrightNoticeDialog.qml b/CopyrightNoticeDialog.qml
index 504bb9b..3a50d5f 100755
--- a/CopyrightNoticeDialog.qml
+++ b/CopyrightNoticeDialog.qml
@@ -62,7 +62,7 @@ Popup {
             id: infoText
             anchors.centerIn: parent
             color: "white"
-            font.pixelSize: 16
+            font.pixelSize: 10
             font.bold: true
             text: qsTr("Copyright (C) 2020-2024\n
 This program is free software: you can redistribute it and/or modify\n
-- 
2.25.1

