From f4572dbaba3879913383117bc3edf25d326f8090 Mon Sep 17 00:00:00 2001
From: beste <1596775839@qq.com>
Date: Tue, 6 Aug 2024 15:08:59 +0800
Subject: [PATCH] FEAT:Add isolcpus=1 in bootargs

---
 include/configs/myd-lma35.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/myd-lma35.h b/include/configs/myd-lma35.h
index 5770796be8..a7c03a3312 100755
--- a/include/configs/myd-lma35.h
+++ b/include/configs/myd-lma35.h
@@ -51,7 +51,7 @@
 	"mmcboot=" \
 	"if mmc dev ${devnum}; then " \
 		"echo \"Booting form mmc ... ...\"; " \
-		"setenv bootargs root=/dev/${mmc_block} rootfstype=ext4 rw rootwait console=ttyS0,115200n8 rdinit=/sbin/init mem=${kernelmem} loglevel=0; " \
+		"setenv bootargs root=/dev/${mmc_block} rootfstype=ext4 rw rootwait isolcpus=1 console=ttyS0,115200n8 rdinit=/sbin/init mem=${kernelmem} loglevel=0; " \
 		"mmc read ${kernel_addr_r} 0x1800 0x8000; " \
 		"mmc read ${fdt_addr_r} 0x1600 0x80; " \
 		"booti ${kernel_addr_r} - ${fdt_addr_r}; " \
@@ -91,7 +91,7 @@
 	"if test ${mtdids} = 'spi-nand0=spi-nand0' ; then " \
 		"echo \"Booting from spinand ... ...\"; " \
 		"mtd list; " \
-		"setenv bootargs noinitrd ubi.mtd=${spinand_ubiblock} root=ubi0:rootfs rootfstype=ubifs rw rootwait=1 console=ttyS0,115200n8 rdinit=/sbin/init mem=${kernelmem} loglevel=1; " \
+		"setenv bootargs noinitrd ubi.mtd=${spinand_ubiblock} root=ubi0:rootfs rootfstype=ubifs rw rootwait=1 isolcpus=1 console=ttyS0,115200n8 rdinit=/sbin/init mem=${kernelmem} loglevel=1; " \
 		"mtd read kernel ${kernel_addr_r}; " \
 		"mtd read device-tree ${fdt_addr_r}; " \
 		"booti ${kernel_addr_r} - ${fdt_addr_r}; " \
-- 
2.25.1

